docs/features/_sidebar.md

```diff
@@ -19,6 +19,7 @@
   * [Config](features/configuration/config.md)
   * [Secrets](features/configuration/secrets.md)
   * [Autoscalers](features/configuration/autoscalers.md)
+  * [Pomerium](features/configuration/pomerium.md)
```

Comment:
`configuration` section is for the Kubernetic > Configuration menu where there is no pomerium.

Move Pomerium documentation to "Enterprise > Single Sign On"